ogc/src/main/scala/geotrellis/server/ogc/wmts/CapabilitiesView.scala

```diff
@@ -160,39 +160,53 @@ object CapabilitiesView {
     )
 
   implicit class OgcSourceMethods(val self: OgcSource) {
-    def toLayerType(layerName: String): LayerType = {
+    def toLayerType(layerName: String, tileMatrixSet: List[GeotrellisTileMatrixSet]): LayerType = {
       val wgs84extent: Extent = ReprojectRasterExtent(self.nativeRE, self.nativeCrs.head, LatLng).extent
+      val tileMatrixLimits: List[TileMatrixLimits] = tileMatrixSet.flatMap(_.tileMatrix.map { tm =>
+        val gridBounds = tm.layout.mapTransform(self.nativeExtent)
```

Comment:
This part seems suspect. I understand `OgcSource.nativeExtent` to be in the projection of the stored `RasterSource` while `TileMatrixSet` can be in arbitrary projection.

Reply:
yes, right; so what projection it should be?